README.0.11.md

```diff
@@ -4,18 +4,17 @@ Terraform Best Practices for AWS users.
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-**Table of Contents**  
 
```

Comment:
This was automatically removed when running `doctoc --github README.0.11.md` command